### PR TITLE
[TC-414] Add NTC info in Nicla Sense ME DS

### PR DESCRIPTION
**Pull request by aliphys**
## What This PR Changes
- Add image showing NTC connector to the Nicla Sense ME datasheet
- Add tables for the ESLOV connector and battery headers
- Add note mentioning the use optional nature of the NTC connector

## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/06.nicla/boards/nicla-sense-me/datasheet/datasheet.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ The Nicla form factor has been specifically developed at Arduino® as a standard
 ### Assembly Overview
 ![Example of a typical solution for remote environmental sensing including an Arduino® Nicla Sense ME, Portenta H7 and battery. Notice the orientation of the battery's cable in the board's connector. ](assets/niclaSenseMEBattery.png)
 
+**Note** : The NTC pin on the battery connector is optional.
```
Comment (TaddyHC) [inline content/hardware/06.nicla/boards/nicla-sense-me/datasheet/datasheet.md:106]:
```suggestion
**Note** : The NTC pin on the battery connector is optional. This is a feature allowing safer use and thermal shutoff of the PMIC. 
```
I checked with the task TC-414 details and thought maybe would be better to provide a little more of detail, and borrowed the word the from the task description. I think it should be enough with the suggested additional detail, but how do you see it?

**Review by TaddyHC — COMMENTED** [review]
One suggestion only is made at the moment. Also, made a small commit which only realigns J5 ESLOV table. I can see we have newly updated render and updated information regarding NTC as described within TC-414 task. Thus it looks good to me. Please have a look at the suggestion.

File: content/hardware/06.nicla/boards/nicla-sense-me/datasheet/datasheet.md (hunk 1 above)
Comment (aliphys) [reply content/hardware/06.nicla/boards/nicla-sense-me/datasheet/datasheet.md:106]: Thanks for bringing this up @TaddyHC . @martab1994 is the suggested change sufficent, or would you like additional information included about the NTC?

Comment (martab1994) [reply content/hardware/06.nicla/boards/nicla-sense-me/datasheet/datasheet.md:106]: That's perfect for me thanks @aliphys and @TaddyHC!

**Review by martab1994 — APPROVED** [review]
LGTM

**Pull request merged**